src/Arcus.Testing.Integration.DataFactory/TemporaryDataFlowDebugSession.cs

```diff
@@ -405,6 +412,42 @@ public RunDataFlowOptions AddDataFlowParameter(string name, object value)
             return this;
         }
 
+        /// <summary>
+        /// Adds a parameter to a DataSet present the DataFlow to run.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="datasetName"/> is blank.</exception>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="name"/> is blank.</exception>
+        public RunDataFlowOptions AddDataSetParameter(string datasetName, string name, object value)
```

Comment:
Can we rename `name` and `value` to `parameterName` and `parameterValue` as to make a clear distiction between the two?